### ISSUE TITLE
t/02.hash_to_field.t and t/09.encode_to_curve.t failures

### ISSUE THREAD

**Issue by eserte** [abbypan/Crypt-OpenSSL-Hash2Curve#1]
I see the following new test failure in Crypt-OpenSSL-Hash2Curve-0.03:
```
#   Failed test 'hash_to_field'
#   at t/02.hash_to_field.t line 19.
#          got: '873DEEBC555053351F80F1E405E6AF44C0AFCD368FA09074F2679DC819861B6E'
#     expected: 'EA083A886A38EF4D15D95BD6A4B4D65620D3C57E4ED00E09FD2D67D67AFD0797'
# Looks like you failed 1 test of 1.
t/02.hash_to_field.t ........................ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
...
#   Failed test 'encode_to_curve'
#   at t/09.encode_to_curve.t line 21.
#          got: '04487A2AB80E2811134AC41C1B7B1B0F5158C523F0A8BBAA9BC5D51DE3BB78B9F2EB728BBAFD8D9436FAA1A4BACCC50DD98ACD53411232E35078162E949B422896'
#     expected: '04993B46E30BA9CFC3DC2D3AE2CF9733CF03994E74383C4E1B4A92E8D6D466B321C4A642979162FBDE9E1C9A6180BD27A0594491E4C231F51006D0BF7992D07127'
# Looks like you failed 1 test of 1.
t/09.encode_to_curve.t ...................... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

Statistical analysis suggests that this error happens if Crypt::OpenSSL::Base::Func 0.033 is installed.

**Comment by eserte** [comment]
With more fail reports statistical analysis suggests something else: it seems that it fails if newer Math::BigInt and bignum is installed.

**Comment by abbypan** [comment]
@eserte  thanks, fixed in Crypt::OpenSSL::Hash2Curve 0.033 , Crypt::OpenSSL::Base::Func 0.036
